problemsets-mat223/modules/module14.tex

```diff
@@ -33,8 +33,15 @@
 	Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{2x+y\\-x+\tfrac{1}{2}y}$.
 	Find the volume of $\mathcal T(C_2)$.
 
-	XXX Finish
+	As we discussed above, $C_2$ is the unit square in $\R^2$ with sides given by $\vec e_1 = \mat{1\\0}$ and $\vec e_2 = \mat{0\\1}$. Apply the linear transformation $\mathcal T$ to $\vec e_1$ and $\vec e_2$, we obtain
+	\[
+	    \mathcal T(\vec e_1)=\mat{2\\-1} \qquad \mathcal T(\vec e_2)=\mat{1\\\frac{1}{2}}.
+	\]
+	Plot $\mat{2\\-1}$ and $\mat{1\\\frac{1}{2}}$ as follows:
```

Comment:
-> `Plotting, we see $\mathcal T(C_2)$ is a parallelogram with base XX and height YY.`